tox.ini

```diff
@@ -47,40 +43,38 @@ commands =
     coverage run setup.py test
     codecov
 
+[testenv:pyre]
+usedevelop=True
+allowlist_externals=
+    cp
+commands =
+    cp .github/workflows/.pyre_configuration .
+    pyre --version
+    pyre check
```

Comment:
This is important to run:
```
python libcst/tests/test_pyre_integration.py
```
circle config has this but github now misses

Reply:
I've added this and now the integration test runs.